tests/kafkatest/services/kafka/config.py

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import property
+
+from sortedcontainers import SortedDict
```

Comment:
Related question is whether this is worth pulling in a library for. Do we use the ordering anywhere other than `render`, and could we accomplish the same just using the built-in `sorted` function? There's no real concern about cost of sorting here, it's just for keeping the code simpler.

Not really an issue to keep it the way it is, it's just another dependency that we're including for very simple functionality.